quickjs-libc.c

```diff
@@ -3026,8 +3084,9 @@ static JSValue js_os_exec(JSContext *ctx, JSValueConst this_val,
             }
         }
 
-        for(i = 3; i < fd_max; i++)
-            close(i);
+        /* close all file descriptors > 2 */
+        close_fds();
+
```

Comment:
You might expand `close_fds()` inline here

Reply:
I completely agree with the redirection to `/dev/null` but not sure about the rest. Overall, this PR is an improvement of current behaviour where we loop through all possible fds (which is inefficient). This PR uses `closefrom` when it is available and mitigate the loop problem when `closefrom` is not supported (which is only on alpine afaik).